apriltag_ros/include/apriltag_ros/continuous_detector.h

```diff
@@ -81,6 +98,24 @@ class ContinuousDetector: public nodelet::Nodelet
 
   ros::ServiceServer refresh_params_service_;
   bool refreshParamsCallback(std_srvs::Empty::Request& req, std_srvs::Empty::Response& res);
+
+  // publishes an empty detection
+  void publishEmptyDetection(const std_msgs::Header& header);
+
+  // helper to convert tf::Vector3 to cv::Point3f
+  cv::Point3f toCvPoint3f(const tf::Vector3& position);
```

Comment:
Move outside the class, or make it `static`:
```suggestion
  static cv::Point3f toCvPoint3f(const tf::Vector3& position);
```

I usually prefer to have methods like this outside the class. You could have it only in the .cpp file, so you don't pollute others scope when they include this header.

Reply:
ok, moved outside of class